File: pandemicBot/src/main/java/com/corona/virus/game/Player.java
package com.corona.virus.game;

public class Player {

    private String id;
    private int score;
    private boolean isCoronned;

    public Player(String id) {
        this.id = id;
        this.score = 0;
        this.isCoronned = false;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public int getScore() {
        return score;
    }

    public void setScore(int score) {
        this.score = score;
    }

    public boolean isCoronned() {
        return isCoronned;
    }

    public void setCoronned(boolean coronned) {
        isCoronned = coronned;
    }

    @Override
    public String toString() {
        return "Nom : " + id + ", score=" + score + "\n";
    }
}

File: pandemicBot/src/main/kotlin/com/corona/virus/game/Player.kt
package com.corona.virus.game

import org.slf4j.LoggerFactory
import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document
import javax.validation.constraints.NotEmpty

@Document
data class Player(@field: NotEmpty(message = "id.required") @Id var id: String) {

    var playerName: String = ""
    var score = 0
    var isCoronned = false

    override fun toString(): String {
        return "playerName: $playerName, score=$score\n"
    }


}